checker/votour.ml

```diff
@@ -100,6 +100,7 @@ struct
             init_size seen (fun n -> fold (succ i) (accu + 1 + n) k) os.(i)
         in
         fold 0 1 (fun size -> let () = LargeArray.set !sizes p size in k size)
+      | Int63 _ -> () (* FIXME *)
```

Comment:
@ppedrot I believe @vbgl was wondering what to put here.

Reply:
This should be `k n` where `n` is the number of words taken by an `Int63` in memory. I guess you can underapproximate by `k 0`, it cannot really hurt.